lib/utils/CHORDTelescope.cpp

```diff
@@ -667,7 +670,7 @@ freq_id_t CHORDTelescope::to_freq_id(stream_t, uint32_t) const {
 
 // TODO: This is a stub to satisfy inheritance and must be updated.
```

Comment:
Is this really a stub? If so, why not abort instead? Or is this real code with a wrong comment?